doc/reference.md

```diff
@@ -1321,7 +1323,9 @@ if it is not.
 ```clojure
  (assoc my-map
    :key1 :val1 :key2 :val2)
-``` #### :arg1-pair-body
+```
+
+#### :arg1-pair-body
```

Comment:
It's description is also probably wrong, looks like a copy-paste from the `:arg1-pair` above.